modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -343,6 +351,11 @@ public void tearDown() throws Exception {
 		// reference a required list type entry cannot be deleted before it is
 		// unreferenced
 
+		PropsUtil.addProperties(
+			UnicodePropertiesBuilder.setProperty(
+				"feature.flag.LPS-171440", "false"
+			).build());
+
```

Comment:
@p-albuquerque please add this before the comment.. since the comment is only related to the `deleteObjectDefinition`